src/acquisition/covidcast/csv_importer.py

```diff
@@ -78,6 +83,12 @@ def find_csv_files(scan_dir, glob=glob):
     time_value) (otherwise None).
```

Comment:
suggest updating this to reflect the new `issue_value` and `lag_value` values returned